devito/types/dimension.py

```diff
@@ -536,6 +535,81 @@ def _arg_check(self, *args, **kwargs):
 # The Dimensions below are exposed in the user API. They can only be created by
 # the user
 
+class SubDimensionThickness(DataSymbol):
+    """A DataSymbol to represent a thickness of a SubDimension"""
```

Comment:
full stop

Reply:
Thought we didn't do those on single line docstrings?